plugins/catalog-backend-module-unprocessed/src/UnprocessedEntitiesModule.ts

```diff
@@ -35,7 +35,7 @@ export class UnprocessedEntitiesModule {
 
   constructor(
     private readonly database: Knex,
-    private readonly router: HttpRouterService,
+    private readonly router: Pick<HttpRouterService, 'use'>,
```

Comment:
This feels really weird to me. The existing interface being compatible with and `express.Router` feels like a really nice nice to have, but I can't think of any other way of attaching the middleware in the router setup or lifecycle that scales.